packages/status-js/src/client/client.ts

```diff
@@ -56,34 +60,30 @@ class Client {
 
   static async start(options: ClientOptions) {
     // Waku
-    const fleet =
-      options.environment === 'test'
-        ? discovery.predefined.Fleet.Test
-        : discovery.predefined.Fleet.Prod
+    const fleet = options.environment === 'test' ? Fleet.Test : Fleet.Prod
     /**
      * >only connects to 1 remote node because of the limited number of nodes
      * >run by Status and the limited number of connections provided by these nodes
      * >
      * >@see https://forum.vac.dev/t/waku-v2-scalability-studies/142/2
      */
     const peers = getPredefinedBootstrapNodes(fleet)
-    const waku = await Waku.create({
-      bootstrap: {
-        default: false,
-        peers,
-      },
+    const waku = await createWaku({
+      defaultBootstrap: false,
+      emitSelf: true,
```

Comment:
This needs to be passed to `WakuRelay`. You may have to use the `Waku` constructor directly, until I have provided more helper functions.

e.g. https://github.com/waku-org/js-waku-examples/blob/f698d97013fc8d1088980706b9fbfe13a1b0b0d2/web-chat/src/App.tsx#L210

Reply:
ditto https://github.com/status-im/status-web/pull/307#discussion_r962932432